templates/frame.haml

```diff
@@ -367,6 +370,26 @@
 
   - block navbar
     - if org
+      -if org|check_policy:"cookie" or org|check_policy:"privacy" or org|check_policy:"tos"
+        %div#ureport-policies-bar{ style: 'border-color: {{ org|config:"primary_color" }}' }
+          %div.content
+            - trans "This website uses cookies to give you an incredible experience. By using this website you agree to the"
+            -if org|check_policy:"cookie"
+              %a{ href:"{% url 'public.policies' 'cookie' %}" }
+                - trans "Cookie Policy"
+              ,
```

Comment:
In the case of having just Cookie Policy, we would have "Cookie Policy," at the end of the text.

Reply:
Updated, please test again.